projects/design-system-angular/src/lib/accordion/accordion.component.ts

```diff
@@ -9,40 +9,42 @@ import {
 import {prefix as PREFIX} from '../config';
 import {ARIA_EXPANDED} from '../utility-functions';
 
-const ACCORDION_SELECTOR = `.${PREFIX}-accordion`;
-const MULTISELECTABLE = 'aria-multiselectable';
+const        ACCORDION_SELECTOR                   = `.${PREFIX}-accordion`;
+const        ACCORDION_BUTTON_SELECTOR            = `.${PREFIX}-accordion__button`;
+export const ACCORDION_MULTISELECTABLE_CLASSNAME  = `${PREFIX}-accordion-multiselectable`;
+export const ACCORDION_CONTENT_EXPANDED_CLASSNAME = `${PREFIX}-accordion__content--expanded`;
 
 export const ARIA_CONTROLS = 'aria-controls';
-export const HIDDEN = 'hidden';
+export const HIDDEN        = 'hidden';
 
 /**
  * An Accordion is a vertically stacked set of headings that each control the visibility of an associated content section.
  *
  * <example-url>http://localhost:4200/jazz-design-system/#/accordion/accordionExample</example-url>
  * @example
-   `` `
```

Comment:
Why did you remove the indentation, this makes it harder to read if you remove it.

Reply:
For an unknown default setting, my Intellij auto-formats spacings on save for some things. I'll add the spacings back in.